### ISSUE TITLE
clients/node doesn't work with vite-node

### ISSUE THREAD

**Issue by akm** [akm/goa_todo_example#5]
## with `ts-node`

```
$ cd clients/nodejs
$ npx ts-node src/todos/create "make check list" open 
{
  id: 2,
  title: 'make check list',
  state: 'open',
  createdAt: '2023-10-28T21:14:58+09:00',
  updatedAt: '2023-10-28T21:14:58+09:00'
}
```

## with `vite-node`

```
$ npx vite-node src/todos/create "make check list" open
Error: 13 INTERNAL: Request message serialization failure: Expected argument of type todos.CreateRequest
    at callErrorFromStatus ($HOME/workspace/goa_todo_example/clients/nodejs/node_modules/@grpc/grpc-js/src/call.ts:82:17)
    at Object.onReceiveStatus ($HOME/workspace/goa_todo_example/clients/nodejs/node_modules/@grpc/grpc-js/src/client.ts:360:55)
    at Object.onReceiveStatus ($HOME/workspace/goa_todo_example/clients/nodejs/node_modules/@grpc/grpc-js/src/client-interceptors.ts:458:34)
    at Object.onReceiveStatus ($HOME/workspace/goa_todo_example/clients/nodejs/node_modules/@grpc/grpc-js/src/client-interceptors.ts:419:48)
    at $HOME/workspace/goa_todo_example/clients/nodejs/node_modules/@grpc/grpc-js/src/resolving-call.ts:132:24
    at processTicksAndRejections (node:internal/process/task_queues:77:11)
for call at
    at ServiceClientImpl.makeUnaryRequest ($HOME/workspace/goa_todo_example/clients/nodejs/node_modules/@grpc/grpc-js/src/client.ts:325:42)
    at ServiceClientImpl.create ($HOME/workspace/goa_todo_example/clients/nodejs/node_modules/@grpc/grpc-js/src/make-client.ts:189:15)
    at $HOME/workspace/goa_todo_example/clients/nodejs/src/todos/create.ts:26:8
    at ViteNodeRunner.runModule (file://$HOME/.npm/_npx/f2342a4b64a2bc92/node_modules/vite-node/dist/client.mjs:342:5)
    at ViteNodeRunner.directRequest (file://$HOME/.npm/_npx/f2342a4b64a2bc92/node_modules/vite-node/dist/client.mjs:326:5)
    at ViteNodeRunner.cachedRequest (file://$HOME/.npm/_npx/f2342a4b64a2bc92/node_modules/vite-node/dist/client.mjs:189:14)
    at ViteNodeRunner.executeFile (file://$HOME/.npm/_npx/f2342a4b64a2bc92/node_modules/vite-node/dist/client.mjs:161:12)
    at CAC.run (file://$HOME/.npm/_npx/f2342a4b64a2bc92/node_modules/vite-node/dist/cli.mjs:91:5) {
  code: 13,
  details: 'Request message serialization failure: Expected argument of type todos.CreateRequest',
  metadata: Metadata { internalRepr: Map(0) {}, options: {} }
}
```

**Comment by akm** [comment]
related articles?

- https://github.com/googleapis/nodejs-tasks/issues/606
- https://www.reddit.com/r/grpc/comments/15og17f/js_serialization_failure/
- https://juejin.cn/post/6949823818097492005

**Comment by akm** [comment]
Fixed by #6
